lib/pactflow/client/provider_contracts/publish.rb

```diff
@@ -22,8 +22,27 @@ def initialize(params, options, pact_broker_client_options)
 
         def do_call
           create_branch_version_and_tags
-          create_contract
-          PactBroker::Client::CommandResult.new(true, green("Successfully published provider contract for #{provider_name} version #{provider_version_number} to Pactflow"))
+          render_response(create_contract)
+        end
+
+        def render_response(res)
+          if res.body && res.body['_links'] && res.body['_links']['pf:ui']['href']
```

Comment:
Try `res._link('pf:ui')&.href`

Reply:
hmm tried that and no bueno, would it be `.links` (also no bueno)

Reply:
will fix forward, and maybe learn some ruby in the process 😅

Reply:
`res.body&.dig("_links", "pf:ui", "href")`